src/metadata/a48.json

```diff
@@ -1239,40 +1239,50 @@
     "key": "fullName"
   },
   {
-    "directoryName": "outboundMessages",
+    "directoryName": "workflowOutboundMessages",
     "inFolder": false,
     "metaFile": false,
-    "suffix": "outboundMessage",
+    "suffix": "workflowOutboundMessage",
     "parentXmlName": "Workflow",
     "xmlName": "WorkflowOutboundMessage",
     "xmlTag": "outboundMessages",
     "key": "fullName"
   },
   {
-    "directoryName": "rules",
+    "directoryName": "workflowRules",
     "inFolder": false,
     "metaFile": false,
-    "suffix": "rule",
+    "suffix": "workflowRule",
     "parentXmlName": "Workflow",
     "xmlName": "WorkflowRule",
     "xmlTag": "rules",
     "key": "fullName"
   },
   {
-    "directoryName": "knowledgePublishes",
+    "directoryName": "workflowSends",
+    "inFolder": false,
+    "metaFile": false,
+    "suffix": "workflowSend",
+    "parentXmlName": "Workflow",
+    "xmlName": "WorkflowSend",
+    "xmlTag": "send",
+    "key": "fullName"
+  },
+  {
+    "directoryName": "workflowKnowledgePublishs",
     "inFolder": false,
     "metaFile": false,
-    "suffix": "knowledgePublishe",
+    "suffix": "workflowKnowledgePublishe",
```

Comment:
Yep, I noticed when propagating this to the beta v6 version.
I fixed it [there](https://github.com/scolladon/sfdx-git-delta/pull/924/files#diff-af5ae85b2e274ae947fd573fb273887862f20536a3f6a2c73729300ad01f86fdR1027).

I'll fix it soon and It will be ok for january 7 2025

Reply:
Sounds good to me!

Reply:
Here is the [PR](https://github.com/scolladon/sfdx-git-delta/pull/944)